apps/chat/src/components/Chat/ChatMessage/AdjustedTextarea.tsx

```diff
@@ -41,10 +59,10 @@ export const AdjustedTextarea = React.forwardRef((props: Props, ref) => {
         value={value}
       />
       <textarea
+        {...restProps}
```

Comment:
Please, revert this to previous place to not override custom `data-qa` (reason of failed e2e-tests)